Policies/Kubernetes/container-no-privilege-escalation/examples/example-bad.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Thinking more about examples...
1. Can you please create two folders "examples-good" and "examples-violations"?
2. If we have more than one good or bad files, we can place them in their folders. For container resource limits, I can think of multiple examples like limits not specified in YAML, limits not within specified limits, limits specified in incorrect format etc.
3. Also when we do end-to-end automation, it will be easy to distinguish which files result in violations.

Reply:
Do we want all possible violation covered? Shouldn't the rego unit tests be used for that?